src/main/resources/application.yml

```diff
@@ -1,3 +1,13 @@
 micronaut:
   application:
     name: ontology
+    version: 0.1
+storeOntology:
+  bucketSubdirectory: ontology
+  configurationFileName: /configuration
+elasticSearch:
+  servers:
+    - elasticsearch5a1.dsde-dev.broadinstitute.org
+    - elasticsearch5a2.dsde-dev.broadinstitute.org
+    - elasticsearch5a3.dsde-dev.broadinstitute.org
```

Comment:
For this app, we should be using the local ES instance. At this point in time, we don't want to be referencing the live dev servers. In the future, I anticipate that this will be a runtime configuration that allows the user to generate an index against any server.